modules/installation-user-defined-tags-azure.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+// * installing/installing_azure/installing-azure-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-azure-user-defined-tags_{context}"]
+= Configuring the user-defined tags for Azure
+
+In {product-title} you can use the tags for grouping resources and for managing resource access and cost. You can define the tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined tags after cluster creation.
+
+Support for user-defined tags is available only in {product-title} 4.13 as a Technology Preview (TP) and for the resources created in the Azure Public Cloud. User-defined tags are not supported for the {product-title} clusters upgraded to {product-title} 4.13.
```

Comment:
This is a complex sentence. Can you try to break it down? If not, you can try to reword something like this suggestion.

```suggestion
Support for user-defined tags is available only for the resources created in the Azure Public Cloud, and in {product-title} 4.13 as a Technology Preview (TP) . User-defined tags are not supported for the {product-title} clusters upgraded to {product-title} 4.13.
```

Reply:
Addressed